static-analysis/teammates-eslint.yml

```diff
@@ -139,7 +139,7 @@ rules:
     guard-for-in: "error"
 
     # disallow the use of alert, confirm, and prompt
-    no-alert: "off" # recommended by Google
+    no-alert: 2 # recommended by Google
```

Comment:
To standardize, let's make it `"error"` instead of `2`, and remove the "recommended by Google" comment.